lib.bzl

```diff
@@ -27,6 +27,8 @@ load("//lib:types.bzl", _types = "types")
 load("//lib:unittest.bzl", _asserts = "asserts", _unittest = "unittest")
 load("//lib:versions.bzl", _versions = "versions")
 
+# There isn't a better way to issue warnings in Starlark, so use print.
+# buildifier: disable=print
```

Comment:
use fail and remove the rest of the file

Reply:
Done